src/test/java/org/ice4j/util/DatagramPacketBufferTest.java

```diff
@@ -34,7 +34,7 @@ public void testDatagramPacketOffsetAndLengthRespected() throws Exception
         // holds flag if all checks were succeeded.
         final AtomicBoolean allChecksSucceeded = new AtomicBoolean(true);
 
-        final int MAX_SEND_DATAGRAM_SIZE = 256;
+        final int maxSendDatagramSize = 256;
```

Comment:
I think the preference is for capital letters in constants

Reply:
Ok, just thought that scope of this constant is a method where lost of other final variables declared. No problem change this back.